### PR TITLE
Mention overriding soft-mandatory policies in CLI

### PR DESCRIPTION
**Pull request by rberlind**
Users can override soft-mandatory Sentinel policies when using the Terraform CLI with the remote backend. They are shown the following:
```
Do you want to override the soft failed policy check?
  Only 'override' will be accepted to override.

  Enter a value:
```

They must type "override" to override.  Note that "yes" will not work.

## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout

### CODE REVIEW

File: content/source/docs/cloud/run/cli.html.md [hunk 1]
```diff
@@ -182,7 +182,39 @@ Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
 
 ## Sentinel Policies
 
-If the specified workspace uses Sentinel policies, those policies will run against all speculative plans and remote applies in that workspace. The policy output will be available in the terminal. Hard mandatory checks cannot be overridden and they prevent `terraform apply` from applying changes.
+If the specified workspace uses Sentinel policies, those policies will run against all speculative plans and remote applies in that workspace. The policy output will be available in the terminal. Soft mandatory checks can be overridden by users with the [Manage Policies](docs/cloud/users-teams-organizations/permissions.html#manage-policies) permission by typing "override" in response to the prompt. (Note that "yes" will not work.)
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/cli.html.md:185]:
```suggestion
If the specified workspace uses Sentinel policies, those policies will run against all speculative plans and remote applies in that workspace. Policy output is shown in the terminal. 

Failed policies can pause or prevent an apply, depending on the enforcement level:

- Hard mandatory checks cannot be overridden and they prevent `terraform apply` from applying changes.
- Soft mandatory checks can be overridden by users with permission to [manage policies](docs/cloud/users-teams-organizations/permissions.html#manage-policies). If your account can override a failed check, Terraform will prompt you to type "override" to confirm. (Note that typing "yes" will not work.) If you override the check, you will be prompted to apply the run (unless auto-apply is enabled).

[permissions-citation]: #intentionally-unused---keep-for-maintainers
```

File: content/source/docs/cloud/run/cli.html.md [hunk 2]
```diff
@@ -182,7 +182,39 @@ Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
...
+FALSE - my-policy.sentinel:1:1 - Rule "main"
+
+Do you want to override the soft failed policy check?
+  Only 'override' will be accepted to override.
+
+  Enter a value: override
+```
+If you do override, you will then be prompted to apply the run. 
+
+Hard mandatory checks cannot be overridden and they prevent `terraform apply` from applying changes.
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/cli.html.md:217]:
```suggestion
```

**Review by nfagerlund — CHANGES_REQUESTED** [review]
Thanks for this! 

I went into the preview to evaluate this. The two code blocks that show policy check output are too similar to both be valuable, and they make the section too visually noisy. 

Could you please remove the second code block? That way we'll show the more complicated case (the override prompt), and readers can just intuit that the simpler case looks much the same but without the option to interact.

I also have some suggestions about changing the ordering of information a bit, to reduce the amount of stuff the reader needs to remember while they skim the code block.

**Comment by rberlind** [comment]
@nfagerlund : I committed your proposed revisions and removed the second Sentinel policy failure example.  I think that should cover all your proposed changes.

**Review by nfagerlund — APPROVED** [review]
Indeed!

**Pull request merged**